### PR TITLE
handler

### PR DESCRIPTION
**Pull request by jinyangyang222**

### CODE REVIEW

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/enums/DeployResourceKind.java [hunk 1]
```diff
@@ -15,10 +15,11 @@
...
 public enum DeployResourceKind {
-    Vm("vm"),
-    Container("container"),
-    PublicIp("public_ip"),
-    Vpc("vpc");
+    VM("vm"),
+    CONTAINER("container"),
+    PUBLICIP("public_ip"),
+    VPC("vpc"),
+    DISK("disk");
```
Comment (iskey) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/enums/DeployResourceKind.java]: I suggest renaming the disk to volume.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/Vm.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * Vm model.
+ */
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@Data
+public class Vm extends DeployResource {
+
+    public String ip;
+    public String region;
```
Comment (iskey) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/Vm.java]: I think the region should not be the property of VM.
`cpu`,`mem` should be added.

**Review by iskey — CHANGES_REQUESTED** [review]
comments added.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/enums/DeployResourceKind.java (hunk 1 above)
Comment (jinyangyang222) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/enums/DeployResourceKind.java:22]: ok, I have modified

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/Vm.java (hunk 2 above)
Comment (jinyangyang222) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/Vm.java:23]: region removed

**Pull request closed without merging**